CHANGELOG.md

```diff
@@ -1,3 +1,236 @@
+- Add check for multiple release in Namespace
+  ([#1093](https://github.com/kinvolk/lokomotive/pull/1093)).
+
+#### Development
+
+- fixed error capitalization
+  ([#979](https://github.com/kinvolk/lokomotive/pull/979)).
+- pkg/terraform: unexport functions not used outside of package
+  ([#984](https://github.com/kinvolk/lokomotive/pull/984)).
+- pkg/platform: fix imports order
+  ([#983](https://github.com/kinvolk/lokomotive/pull/983)).
+- pkg/components: remove unused List() function
+  ([#982](https://github.com/kinvolk/lokomotive/pull/982)).
+- components/metallb: Add missing autodiscovery labels
+  ([#990](https://github.com/kinvolk/lokomotive/pull/990)).
+- pkg/asssets/assets_generate: Fix copyright
+  ([#1020](https://github.com/kinvolk/lokomotive/pull/1020)).
+- Cleanup Terraform providers before Terraform 0.13 upgrades
+  ([#860](https://github.com/kinvolk/lokomotive/pull/860)).
+- kubelet e2e: Enable the disruptive test
+  ([#1012](https://github.com/kinvolk/lokomotive/pull/1012)).
+- .golangci.yml: Re-enable linters
+  ([#1029](https://github.com/kinvolk/lokomotive/pull/1029)).
+- Fix scripts/find-updates.sh
+  ([#1034](https://github.com/kinvolk/lokomotive/pull/1034)),
+  ([#1068](https://github.com/kinvolk/lokomotive/pull/1068)),
+  ([#1080](https://github.com/kinvolk/lokomotive/pull/1080)).
+- pkg/terraform: improvements
+  ([#1027](https://github.com/kinvolk/lokomotive/pull/1027)).
+- cli/cmd: cleanups part 1
+  ([#1013](https://github.com/kinvolk/lokomotive/pull/1013)).
+- test/components/kubernetes: remove kubelet pod when testing node labels
+  ([#1052](https://github.com/kinvolk/lokomotive/pull/1052)).
+- remove usage of template_file
+  ([#1046](https://github.com/kinvolk/lokomotive/pull/1046)).
+- test/*: de-duplicate value timeout and retryInterval
+  ([#1049](https://github.com/kinvolk/lokomotive/pull/1049)).
+- packet: Read BGP peer address from metadata service
+  ([#1010](https://github.com/kinvolk/lokomotive/pull/1010)).
+- pkg/assets: cleanup exported API
+  ([#936](https://github.com/kinvolk/lokomotive/pull/936)).
+- cobra upgrade to v1.1.1
+  ([#1082](https://github.com/kinvolk/lokomotive/pull/1082)),
+  ([#1091](https://github.com/kinvolk/lokomotive/pull/1091)).
+- Update grafana and other prometheus operator sub components
+  ([#963](https://github.com/kinvolk/lokomotive/pull/963)).
+- e2e: Add test for prometheus-operator PVC
+  ([#1087](https://github.com/kinvolk/lokomotive/pull/1087)).
+- cli/cmd: cleanups part 2
+  ([#1015](https://github.com/kinvolk/lokomotive/pull/1015)).
+- Add github actions
+  ([#1074](https://github.com/kinvolk/lokomotive/pull/1074)).
+- Makefile: use latest Go when building in Docker
+  ([#1083](https://github.com/kinvolk/lokomotive/pull/1083)).
+- Rename Headlamp to Web UI and add OIDC support
+  ([#1100](https://github.com/kinvolk/lokomotive/pull/1100)).
+- cli/cmd: cleanups part 3
+  ([#1018](https://github.com/kinvolk/lokomotive/pull/1018)).
```

Comment:
I'd trim this section down to essentials. Ones that matter when the reader reads this changelog.